scripts/js2cpp/build.js

```diff
@@ -123,8 +123,7 @@ function GenerateCPP(platform, buildDirPath) {
         code += CodePieces.piece_1;
 
         for(let i = 2; i < files_arr.length; i++) {
-            let file_split_arr = files_arr[i].split('/');
-            let file_name = file_split_arr[file_split_arr.length - 1].split('.')[0];
+			  let file_name = path.basename(files_arr[i], ".js")
```

Comment:
It seems indent have mistaken.

Reply:
ok i have change the indent to the origin codes indent .